schema.sql

```diff
@@ -64,8 +64,17 @@ CREATE TABLE specializations (
 CREATE TABLE visits (
     id_vet INT,
     id_animal INT,
-    date_of_visit DATE, 
-    PRIMARY KEY (id_vet, id_animal, date_of_visit),
+    date_of_visit DATE,     
     FOREIGN KEY(id_vet) REFERENCES vets (id), 
     FOREIGN KEY(id_animal) REFERENCES animals (id)
-);
\ No newline at end of file
+);
+
+-- Add an email column to your owners table
+ALTER TABLE owners ADD COLUMN email VARCHAR(120);
+
+-- Create Index on the visits and owners table for persormance and lesser excution time
+CREATE INDEX id_animal_asc ON visits(id_animal ASC);
+CREATE INDEX vet_ind ON visits(id_vet ASC);
+CREATE INDEX email_index ON owners(email);
```

Comment:
- [ ] Kindly make the execution time for the third screenshot before the optimization exceeds or equal `1000ms` as per this [requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_performance_audit.md#:~:text=4%20Depening%20on,bigger%20than%201000ms), Currently, it's only `811.200ms`.

Reply:
@gemmen29 thanks for the review, I will update the pull request with your request.